### PR TITLE
Finished benchmark fix

### PR DESCRIPTION
**Pull request by bartosh**

### CODE REVIEW

**Comment by tibkiss** [comment]
Thanks @bartosh for this PR.

For the record: This PR moves from Yahoo based ^GSPC benchmark to Google based SPY benchmarking. This change is required due to the recent unadopted Yahoo Finance API change which breaks the downloads through Pandas.

File: zipline/data/benchmarks.py [hunk 1]
```diff
@@ -24,5 +25,14 @@ def get_benchmark_returns(symbol, start_date, end_date):
     start_date is **not** included because we need the close from day N - 1 to
     compute the returns for day N.
     """
-    df = web.DataReader(symbol, 'google', start_date, end_date)
-    return df["Close"].sort_index().tz_localize('UTC').pct_change(1).iloc[1:]
+    df = web.DataReader(symbol, 'google', start_date, end_date) 
+    df.index = df.index.tz_localize('UTC')
+
+    calendar = get_calendar("NYSE")
+    start_index = calendar.all_sessions.searchsorted(start_date)
```
Comment (tibkiss) [inline zipline/data/benchmarks.py]: Could you please explain why indexing is required here? The original code seemed to work well with start_date and end_date supplied to DataReader.

Comment (bartosh) [inline zipline/data/benchmarks.py]: As I mentioned in the commit message Google Finance price data for SPY missing some dates, for example 2012-02-02, which causes algorithms to break with
    KeyError: 'the label [2012-02-02 00:00:00+00:00] is not in the [index]'

**Comment by tibkiss** [comment]
Thanks for this fix, Ed!

**Pull request merged**